CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
 Change Log
 ==========
 
+## Version 1.6.0
+
+_2020-05-28_
+
+ * New: Deprecate Mirror API integrations.
+ 
+   Mirror API integrations, such as `TypeElement.asClassName()` and 
```

Comment:
@JakeWharton @ZacSweers can you please fact-check this paragraph?

Reply:
Lgtm! Maybe we should mention it's soft deprecated for now and we may explore metadata-based alternatives later?

Reply:
Makes sense, wanna reword in a followup PR?